parsers/US_ERCOT.py

```diff
@@ -0,0 +1,366 @@
+"""Parser for the ERCOT grid area of the United States."""
+
+import gzip
+import json
+import zipfile
+from datetime import datetime, timedelta, timezone
+from io import BytesIO
+from logging import Logger, getLogger
+from zoneinfo import ZoneInfo
+
+import arrow
```

Comment:
We are trying to remove arrow from the parsers so would be great if we could avoid adding it here.